CONTRIBUTING.md

```diff
@@ -24,7 +24,7 @@ Contributing follows this workflow:
 
 1. Fork [this project repository](https://github.com/codecademy/docs).
 2. Clone the forked repository to your computer.
-3. Create a branch.
+3. Create, and switch into, a new branch.
```

Comment:
Are these commas required or is it still grammatically correct without them?

Reply:
Either way seems grammatically okay because both distinct actions (creating and switching into) relate to one specific target (the new branch). I'll remove for better readability.